packages/Compute/src/package.ts

```diff
@@ -227,3 +227,16 @@ export function modelCatalog() {
     }
   } else grok.shell.v = modelsView;
 }
+
+//name: RangeValidatorFactory
+//input: object params
```

Comment:
Such input types are not supported outside of JS functions - but I think, it is not a problem.
1. Python/Octave/etc, validators are generally bad idea - they could not be fast
2. If smth outside of JS functions will be required - we easily can call Python functions right from JS validation functions (ofc, if we support async validators)

Noting that to be sure that we are on the same page regarding this question

Reply:
Don't know...
This `object` type could be tricky to implement for all languages supported. 

Anyway, the main idea for now is "use JS validators everywhere you can"

Reply:
>If smth outside of JS functions will be required - we easily can call Python/Octave/etc. functions right from JS validation functions (ofc, if we support async validators) 

Exactly!